### ISSUE TITLE
Intent of pressing `q` when exiting editor with unsaved buffers?

### ISSUE THREAD

**Issue by bjornbm** [philanc/ple#6]
In `ple.lua`, line 733+, in the `exiteditor` function, we find:

```lua
	if anyunsaved then
		local ch = readchar(
			"Some buffers not saved. Quit? ", "[YNQynq\r\n\24]")
		if ch ~= "y" and ch ~= "Y" then
			msg("aborted.")
			return
		end
	end
```

I just want to double-check whether the intent of pressing `q` is as implemented? As implemented, pressing `q` aborts the quit (same answering `n` to the question “Quit?”). This is not the obvious behavior to me, although I guess I can make sense in terms of “quitting the on-going operation” in a general sense, disregarding that the operation happens to be to quit the editor.

To sidestep the question and avoid possible confusion/unexpected behavior, I would probably just remove 'Q' and 'q' from the pattern. 😜

Let me know if you want a patch for this.

**Comment by philanc** [comment]
Agreed.  Will just remove 'Qq' from the pattern. Thanks.

**Comment by bjornbm** [comment]
Closed by 2debb10
